File: output/ChatGPT/codenet/Java/C++/s302365683.cpp
#include <iostream>
#include <unordered_map>
using namespace std;

int main() {
    int n;
    cin >> n;
    
    unordered_map<int, int> hm;
    for (int i = 0; i < n; i++) {
        int val;
        cin >> val;
        hm[val]++;
    }
    
    int ans = 0;
    for (auto it : hm) {
        int num = it.second;
        if (num > it.first)
            ans += num - it.first;
        else if (num < it.first)
            ans += num;
    }
    
    cout << ans << endl;
    
    return 0;
}

File: output/ChatGPT/codenet/Java/Python/s302365683.py
n=int(input())
hm={}
arr=list(map(int,input().split()))
for val in arr:
    hm[val]=hm.get(val,0)+1
ans=0
for val in hm.keys():
    num=hm[val]
    if num>val:
        ans+=num-val
    elif num<val:
        ans+=num
print(ans)